index.html

```diff
@@ -1,174 +1,285 @@
-                            </ul> 
-                    </article>   
-                </section>
+    <article class="aboutme">
+      <h2>About me</h2>
+      <p class="describeme">
+        I can help you build a product ,<br />
+        feature or website Look through <br />
+        some of my work and experience! If <br />
+        you like what you see and have a<br />
+        project you need coded, <br />
+        don’t hestiate to contact me.
+      </p>
+      <button class="resumebutton buttons" type="submit">Get My Resume</button>
+    </article>
+    <section class="grid-container">
+      <article class="aboutmebox">
+        <img src="img/languages.png" alt="language details" />
+        <p class="describeaboutmebox">Languages</p>
+        <ul>
+          <li><a href="#">Javascript</a></li>
+          <li><a href="#">Ruby</a></li>
+          <li><a href="#">Html</a></li>
+          <li><a href="#">CSS</a></li>
+        </ul>
+      </article>
+      <article class="aboutmebox">
+        <img src="img/frameworks.png" alt="frameworks details" />
+        <p class="describeaboutmebox">Frameworks</p>
+        <ul>
+          <li><a href="#">Bootstrap</a></li>
+          <li><a href="#">Ruby on rails</a></li>
+          <li><a href="#">Rspec</a></li>
+          <li><a href="#">Capybara</a></li>
+          <li><a href="#">Selenium</a></li>
+        </ul>
+      </article>
+      <article class="aboutmebox">
+        <img src="img/skills.png" alt="skills details" />
+        <p class="describeaboutmebox">Skills</p>
+        <ul>
+          <li><a href="#">Codekit</a></li>
+          <li><a href="#">Github</a></li>
+          <li><a href="#">Codepen</a></li>
+          <li><a href="#">Gitlab</a></li>
+          <li><a href="#">Terminal</a></li>
+        </ul>
+      </article>
+    </section>
 
+    <footer class="contact-me">
+      <h2>
+        I'm always<br />
+        interested in <br />hearing about<br />
+        new projects, so <br />if you'd like to <br />chat please get<br />
+        in touch.
+      </h2>
 
+      <form action="https://formspree.io/f/xjvznrbv" method="post">
+        <ul class="contact-me-ul">
+          <li>
+            <input
+              type="text"
+              id="name"
+              maxlength="30"
+              name="user_name"
+              required
+              placeholder="fullname"
+            />
+          </li>
+          <li>
+            <input
+              type="email"
+              id="mail"
+              name="user_email"
+              required
+              placeholder="Email address"
+            />
+          </li>
+          <li>
+            <textarea
+              id="msg"
+              name="user_message"
+              maxlength="500"
+              required
+              placeholder="Enter text here"
+            ></textarea>
+          </li>
+          <li>
+            <input type="submit" value="Get in touch" class="btn buttons" />
```

Comment:
Kindly add interaction such as a hover effect to your button when clicked just like what we have in the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468&t=W3PwpACp8rxJFwSN-0)👇
<img width="772" alt="Screenshot 2022-12-06 at 21 09 14" src="https://user-images.githubusercontent.com/73854999/206016016-8f718875-e273-44ba-88c0-ee64288fb5b0.png">

Reply:
the hover button is working as indicated on the figma, it was meant to drop a shadow effect on hover.